Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.cs

```diff
@@ -157,6 +160,17 @@ private void UpdateShadowMask()
                 {
                     mask = ((TextBlock)Content).GetAlphaMask();
                 }
+                else if (Content is ImageExBase)
+                {
+                    var imageExBase = (ImageExBase)Content;
+
+                    while (!imageExBase.IsInitialized)
```

Comment:
Is it possible to replace this with an event from imageEx instead of while looping?